openedx/core/djangoapps/course_groups/migrations/0002_change_inline_default_cohort_value.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('course_groups', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='coursecohortssettings',
+            name='always_cohort_inline_discussions',
+            field=models.BooleanField(default=False),
```

Comment:
Note to reviewers: According to documentation, this change will not impact the actual db table at all (so no performance impact).

https://docs.djangoproject.com/en/1.10/ref/migration-operations/#alterfield
`Django never sets database defaults and always applies them in the Django ORM code.`